sunpy/net/dataretriever/sources/tests/test_eve.py

```diff
@@ -117,7 +114,7 @@ def test_levels(time):
 
 def test_attr_reg():
     assert a.Instrument.eve == a.Instrument('EVE')
-    assert a.Level.zero == a.Level('0')
+    assert a.Level('0cs') == a.Level('0cs')
```

Comment:
This test was to make sure the `'0cs'` key is properly processed by the attrs reg code.

Reply:
It was difficult to get alternatives values for registered attrs. Actually in `register_values`, it can take only one possible value, `0` or `0cs`. So I fixed it ``0cs

Reply:
So is there a `a.Level.0cs`?